asv_bench/benchmarks/frame_ctor.py

```diff
@@ -132,7 +132,7 @@ def setup(self, offset, n_steps):
         offset = getattr(offsets, offset)
         self.idx = get_index_for_offset(offset(n_steps, **kwargs))
         self.df = DataFrame(np.random.randn(len(self.idx), 10), index=self.idx)
-        self.d = dict([(col, self.df[col]) for col in self.df.columns])
+        self.d = dict((col, self.df[col]) for col in self.df.columns)
```

Comment:
could have all these be dict comprehensions now that py2.6 is unsupported

Reply:
So you mean `{foo for foo in bar}` instead of `set(foo for foo in bar)`? Sure I don't see why not. Maybe that can be a followup to this.

Reply:
``dict(df.items())`` should work here